### ISSUE TITLE
'allow_user_specified_id' config doesn't work

### ISSUE THREAD

**Issue by junhee-yoo** [edgedb/edgedb-go#249]
**Describe the bug**
Can't create an object with given id when `allow_user_specified_id` configured.

```go
config := make(map[string]interface{})
config["allow_user_specified_id"] = true
cli, err := edgedb.CreateClient(ctx, opts)
cli = cli.WithConfig(config)
```

When I try to insert an object into table, an error occurred:
```shell
QueryError: cannot assign to id
Hint: consider enabling the "allow_user_specified_id" configuration parameter to allow setting custom object ids
```

**Reproduction**
Include the code that is causing the error:

```shell
insert Something {
  id := <uuid>"00000000-0000-0000-0000000000"
}
```

**Expected behavior**
Object should be created with given uuid.

**Versions (please complete the following information):**

-   OS : macOS Monterey (ver.12.6)
-   EdgeDB version: "2.7+784cf0b"
-   EdgeDB CLI version: "EdgeDB CLI 2.2.6+7eabbf9"
-   `edgedb-go` version: "github.com/edgedb/edgedb-go v0.13.0"
-   Go version: go version go1.18.1 darwin/arm64

**Additional context**

This is occurred by: https://github.com/edgedb/edgedb-go/blob/723df88642ba4e19d55989d65e5628c08ec4e2b4/internal/client/transaction.go#L158
which is mis-ordered arguments. newQuery function signature is:
https://github.com/edgedb/edgedb-go/blob/723df88642ba4e19d55989d65e5628c08ec4e2b4/internal/client/query.go#L63-L69

so I'd tried with:
```diff
diff --git a/internal/client/transaction.go b/internal/client/transaction.go
index e754567..fc390ed 100644
--- a/internal/client/transaction.go
+++ b/internal/client/transaction.go
@@ -155,7 +155,7 @@ func (t *Tx) Execute(
 	cmd string,
 	args ...interface{},
 ) error {
-	q, err := newQuery("Execute", cmd, args, t.capabilities1pX(), nil, t.state)
+	q, err := newQuery("Execute", cmd, args, t.capabilities1pX(), t.state, nil)
 	if err != nil {
 		return err
 	}
```
and it worked.

**Comment by fmoor** [comment]
Thanks for pointing this out @junhee-yoo!

**Comment by fmoor** [comment]
this is fixed in v0.13.2

**Comment by junhee-yoo** [comment]
Hi, I'd faced this problem again.

I guess:

https://github.com/edgedb/edgedb-go/blob/014bff79e0ac2fab4886fb1c15c669dea19edcfc/internal/client/transaction.go#L88

this should be fixed as previous one.

**Comment by junhee-yoo** [comment]
@fmoor Sorry for lack of information. And sorry for make you confuse. Maybe I should have made another issue instead of write a comment on closed issue.

According to my investigation, the issue does `not` come from your previous fix `transaction.go:158`. It comes from `transaction.go:88`.

Previously, I got that code link from master branch, but here I got code from v0.13.3 branch:

|line number|code from tag v0.13.3|
|:-:|-|
|L88|https://github.com/edgedb/edgedb-go/blob/20ecc5e449586f96e546cdfd441db7d24819a703/internal/client/transaction.go#L88|
|L158|https://github.com/edgedb/edgedb-go/blob/20ecc5e449586f96e546cdfd441db7d24819a703/internal/client/transaction.go#L158|

I have no idea why this occurs the issue in v0.13.3, because it didn't happen with v0.13.0.
But it seems the arguments order is wrong for `newQuery()` as previous(L158) one and after I fixed it manually, it worked.

Sorry again, and thanks for taking an interest. 👍

**Comment by fmoor** [comment]
Ok thanks! I didn't notice that that was a different line. I pushed a fix and published `v0.13.4`
